Apigee/SmartDocs/Revision.php

```diff
@@ -301,12 +306,15 @@ public function toArray($verbose = TRUE)
      *
      * @param OrgConfig $config
      * @param string $modelId
+     * @param string $modelName
      */
-    public function __construct(OrgConfig $config, $modelId)
+    public function __construct(OrgConfig $config, $modelId, $modelName = "")
     {
         $this->blankValues();
         $this->apiId = $modelId;
-        $this->init($config, '/o/' . rawurlencode($config->orgName) . '/apimodels/' . rawurlencode($this->apiId) . '/revisions');
+        $this->apiName = $modelName;
+//        $this->init($config, '/o/' . rawurlencode($config->orgName) . '/apimodels/' . rawurlencode($this->apiId) . '/revisions');
+      $this->init($config, '/o/' . rawurlencode($config->orgName) . '/apimodels/');
```

Comment:
The init() should pass in the base url which should be /v1/organizations/:org/apimodels/:model/revisions.  If you have methods that use the /v1/organizations/:org/apimodels/ resource, then you need to put those API calls in Model.php